File: Begginers/Python/area_1012.py
linha = input().split()

A = float(linha[0])
B = float(linha[1])
C = float(linha[2])

pi = 3.14159

triangulo = (A * C) / 2
circulo = pi * (C ** 2)
trapezio = ((A + B) * C) / 2
quadrado = B * B
retangulo = A * B

print(f"TRIANGULO: {triangulo:.3f}")
print(f"CIRCULO: {circulo:.3f}")
print(f"TRAPEZIO: {trapezio:.3f}")
print(f"QUADRADO: {quadrado:.3f}")
print(f"RETANGULO: {retangulo:.3f}")

File: Begginers/C++/area_1012.cpp
#include <stdio.h>
 
int main() {
 
    double a, b, c, pi;
    
    pi = 3.14159;
    
    scanf("%lf %lf %lf", &a, &b, &c);
    
    printf("TRIANGULO: %.3lf\n", ((1.0 / 2) * a * c));
    printf("CIRCULO: %.3lf\n", (pi * (c * c)));
    printf("TRAPEZIO: %.3lf\n", ((1.0 / 2) * (a + b) * c));
    printf("QUADRADO: %.3lf\n", (b * b));
    printf("RETANGULO: %.3lf\n", (a * b));
    
    return 0;
}
